commons/src/test/scala/dagr/commons/util/LoggerTest.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015-2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package dagr.commons.util
+
+import java.io.{PrintStream, ByteArrayOutputStream}
+
+import org.scalatest.BeforeAndAfterEach
+
+class LoggerTest extends UnitSpec with LazyLogging with BeforeAndAfterEach {
+
+  private val level: LogLevel = Logger.level
+
+  override def afterEach(): Unit = {
+    Logger.level = level
+  }
+
+  "Logger.sanitizeSimpleClassName" should "display scala classes in a readbale form" in {
+    Logger.sanitizeSimpleClassName(classOf[LoggerTest].getSimpleName) shouldBe "LoggerTest"
+  }
+
+  "Logger" should "log an exception" in {
+    val logger = new Logger(classOf[LoggerTest])
+    val stream = new ByteArrayOutputStream()
+    val pw = new PrintStream(stream)
+    logger.out = Some(pw)
+    val ex = new IllegalArgumentException("Arg!")
+    logger.exception(ex)
+    stream.toString should include ("Arg!")
+  }
+
+  it should "log a message or throwable when the logger is at the same level" in {
+    Logger.synchronized {
```

Comment:
I'll change it to `this.synchronized` but it modifies the `level` value in the `Logger` singleton.  I'll also add a comment:
`// synchronized since we modify the log level, and we don't want to affect other tests`